listenbrainz/webserver/static/js/jsx/follow-users.jsx

```diff
@@ -42,6 +46,43 @@ export class FollowUsers extends React.Component {
     });
   }
 
+  saveFollowList(event) {
+    var listName = this.state.listName;
+    if (this.nameInput.value != undefined && this.nameInput.value != "") {
+      listName = this.nameInput.value;
+    }
+    fetch(this.state.saveUrl, {
+      method: "POST",
+      body: JSON.stringify({
+        "users": this.state.users,
+        "name": listName,
+        "id": this.state.listId,
+      }),
+    })
+    .then(response => response.json())
+    .then(data => {
+      console.debug(data);
+      console.debug("old List ID: " + this.state.listId);
+      this.setState(prevState => {
+        return {listId: data.list_id};
+      });
+      console.debug("new List ID: " + this.state.listId);
+    })
+    .catch(error => {
+        console.error(error);
+    });
+  }
+
+  newFollowList(event) {
+    this.setState(prevState => {
+      return {
+        users: [],
+        listId: null,
+     };
+    });
+    this.nameInput.value = "Untitled Follow List";
```

Comment:
Just set the input value to `null` or an empty string to show the placeholder text (which has a sane default in profile.jsx)

Reply:
the placeholder text is the name of the old list, so if we create a new list, we can't make it show the placeholder text. :sheepish:

Reply:
Ok, fixed now.